docs/ai.md

```diff
@@ -78,3 +78,69 @@ EXAMPLES
 ```
 
 _See code: [src/commands/ai/text-gen.js](https://github.com/box/boxcli/blob/v4.0.1/src/commands/ai/text-gen.js)_
+
+## `box ai:extract`
+
+Sends an AI request to supported LLMs and returns metadata in form of key-value pairs.
+
+```
+USAGE
+  $ box ai:extract --prompt <value> --items <value>... --ai_agent <value> [-t <value>] [--as-user <value>] [--no-color] [--json |
```

Comment:
The flag `ai_agent` is inconsistent with other params, normally we use kebab case for flags, like `no-color` or `as-user`,...